curriculum/challenges/english/02-javascript-algorithms-and-data-structures/regular-expressions/match-characters-that-occur-zero-or-more-times.english.md

```diff
@@ -58,5 +58,9 @@ let result = chewieQuote.match(chewieRegex);
 
 ```js
 // solution required
-```
+```js
+let chewieQuote = "Aaaaaaaaaaaaaaaarrrgh!";
+let chewieRegex = /A[a]*/; // Change this line
+let result = chewieQuote.match(chewieRegex);
+console.log(result); // Aaaaaaaaaaaaaaaa
```

Comment:
console.log(result) is not required to be part of the solution as the challenge does not mention about logging the result